get-resource.sh

```diff
@@ -28,9 +28,16 @@ else
     exit 1
 fi
 
+RHCOS_IMAGE_FILENAME_COMPRESSED=${RHCOS_IMAGE_FILENAME_QCOW/-openstack/-compressed}
 RHCOS_IMAGE_FILENAME_CACHED=cached-${RHCOS_IMAGE_FILENAME_QCOW}
 FFILENAME="rhcos-ootpa-latest.qcow2"
 
+# Added to fix image path incompatibility between 4.7 and 4.8
+# as filed in https://bugzilla.redhat.com/show_bug.cgi?id=1972572
+if [[ -s "/shared/html/images/$RHCOS_IMAGE_FILENAME_QCOW/$RHCOS_IMAGE_FILENAME_COMPRESSED.md5sum" && ! -s "/shared/html/images/$RHCOS_IMAGE_FILENAME_QCOW/$RHCOS_IMAGE_FILENAME_CACHED.md5sum" ]]; then
+    exit 0
```

Comment:
I guess this will work, but would it be better to move `$RHCOS_IMAGE_FILENAME_COMPRESSED` to `$RHCOS_IMAGE_FILENAME_CACHED` - then the filenames become consistent after upgrade (e.g same as a freshly installed 4.8 cluster)?

Reply:
Thanks @hardys.

Do you mean by workaround this change or this one;

https://github.com/openshift/ironic-rhcos-downloader/blob/09c293e6a38e467a084d6182c5830d34f2018951/get-resource.sh#L88

Because if we rename files this workaround seems to be removed.

Reply:
I'm wrong, that does not need to be removed. 

I'll change according to your suggestions.